### DISCUSSION TITLE
https://blog.tidy-intelligence.com/posts/ggplot2-vs-matplotlib/

### DISCUSSION THREAD

**Discussion by giscus** [tidy-intelligence/blog#4] [Announcements]
# https://blog.tidy-intelligence.com/posts/ggplot2-vs-matplotlib/

A comparison of the two most popular data visualization tools for R and Python

https://blog.tidy-intelligence.com/posts/ggplot2-vs-matplotlib/

**Comment by bamattre** [comment]
As someone who knows R and uses ggplot2, I've found it much easier to use Seaborn in Python than matplotlib. I think it's easier to use since it's declarative, and also has nice default theming.

**Reply by christophscheuch** [reply]
If you like {ggplot2} and have to use Python, then there is no way around {plotnine}: https://blog.tidy-intelligence.com/posts/ggplot2-vs-plotnine/. Posit recently started supporting it, so I expect it to become even better maintained soon.
